### ISSUE TITLE
Why total iterations are different to them in the paper?

### ISSUE THREAD

**Issue by nijkah** [kmaninis/OSVOS-PyTorch#25]
in code train_parent.py
The epoch is set to 240 ( **500,000** /2079, 2079 is the number of training images.)
But both implemntation on tensorflow and the paper say total iteration is **50,000.** 
Is there any certain reason to this?

**Comment by kmaninis** [comment]
the factor of 10 regards [this](https://github.com/kmaninis/OSVOS-PyTorch/blob/36562efeed51dd994b44329ea3cff5e0d76e83ac/train_parent.py#L46) line of code. We average the gradients over 10 iterations, so to be more precise there are ~50000 backprops happening.
